src/d.ts

```diff
@@ -81,29 +79,28 @@ export function w<W extends WidgetBaseInterface>(widgetConstructor: Constructor<
  * Wrapper function for calls to create hyperscript, lazily executes the hyperscript creation
  */
 export function v(tag: string, properties: VirtualDomProperties, children?: DNode[]): HNode;
-export function v(tag: string, children: DNode[]): HNode;
+export function v(tag: string, children: undefined | DNode[]): HNode;
 export function v(tag: string): HNode;
-export function v(tag: string, propertiesOrChildren: VirtualDomProperties | DNode[] = {}, children: DNode[] = []): HNode {
+export function v(tag: string, propertiesOrChildren: VirtualDomProperties | DNode[] = {}, children: undefined | DNode[] = undefined): HNode {
 		let properties: VirtualDomProperties = propertiesOrChildren;
 
 		if (Array.isArray(propertiesOrChildren)) {
 			children = propertiesOrChildren;
 			properties = {};
 		}
 
-		let { classes } = properties;
-		if (typeof classes === 'function') {
-			classes = classes();
-			properties = assign(properties, { classes });
+		if (properties) {
```

Comment:
Not sure I can see how properties could be undefined here?

Reply:
Yup, I think could possibly remove that (it's because at one point I allowed undefined properties and changed my mind) - but there is a change following these that will remove this entire section so perhaps wait for that. I don't mind.

Reply:
Updated